README.md

```diff
@@ -232,7 +232,7 @@ Tram-One has a simple interface to help build your web app.
 
 ### `Tram.html([registry])`
 _Reference: [hyperx](https://github.com/substack/hyperx),
-[bel-create-element](https://github.com/JRJurman/bel-create-element),
+[belit](https://github.com/JRJurman/belit),
```

Comment:
Shouldn't this be `Tram-One/belit`?

Reply:
yes, yes it should!... will fix!

Reply:
it has been done ⚡️ 
as well as all the other JRJurman references